### ISSUE TITLE
Suggestion

### ISSUE THREAD

**Issue by dmitriyaa** [FrontendMasters/intro-node-js#20]
I would suggest to add .nvmrc.
`npm i` doesn't work properly for me with node v12

**Comment by KornelijusGlinskas** [comment]
Yeah, I've encountered the same issue. However, with NVM I installed and used the earlier version of node (8.12.0) for this tut and it worked just fine. :)

**Comment by Aj-vrod** [comment]
The current version is 15+ and while running `nvm alias default node` I got issues, once I changed it to 10.9.0 version it worked just fine.
